### PR TITLE
Added more columns to the output (is_SE, self_angry)

### PR DESCRIPTION
**Pull request by sophieball**
Also changed the way I average politeness features (not yet using logistic regression), outputs the politeness features for logistic regression.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,17 +112,23 @@ Before you can run the script, you should first create a file `config.py`, which
 
 ## Data
 
-- Training and testing files should be stored in `src/data/'.
-- Training data (file `src/data/training_data.csv`) should contain at least 3 columns: `_id`, `text`, and `toxic`
-- Test data (file `src/data/test_data.csv`) should contain at least 2 columns: `_id` and `text`
-- The output will be written to `src/classification_results.csv`.
+- Training data should contain at least 3 columns: `id`, `text`, and `label`,
```
Comment (CaptainEmerson) [inline README.md:115]: 4?

File: README.md [hunk 2]
```diff
@@ -112,17 +112,23 @@ Before you can run the script, you should first create a file `config.py`, which
...
+  and a column `training` set to 1 indicating that these data are for training
+  phase.
+- Unlabeled data should contain at least 2 columns: `id` and `text`, and a column
```
Comment (CaptainEmerson) [inline README.md:118]: 3?

File: src/suite.py [hunk 3]
```diff
@@ -489,9 +490,10 @@ def set_parameters(self):
 
   def issue_classifications_from_comments(self):
     t = time.time()
-    self.test_data = self.classify_test()
+    #self.test_data = self.classify_test()
```
Comment (CaptainEmerson) [inline src/suite.py:493]: rm?

**Comment by sophieball** [comment]
I've fixed all of them in PR #33

File: README.md (hunk 1 above)
Comment (sophieball) [reply README.md:115]: Fixed in PR #32

File: README.md (hunk 2 above)
Comment (sophieball) [reply README.md:118]: Fixed in PR #32

File: src/suite.py (hunk 3 above)
Comment (sophieball) [reply src/suite.py:493]: Fixed in PR #32

**Pull request merged**